Formula/gator.rb

```diff
@@ -0,0 +1,20 @@
+class Gator < Formula
+  desc "CLI Utility for Open Policy Agent Gatekeeper"
+  homepage "https://open-policy-agent.github.io/gatekeeper/website/docs/gator"
+  url "https://github.com/open-policy-agent/gatekeeper.git",
+    tag:      "v3.9.0",
+    revision: "dfd2dd80734e20d30c00f7e729f03ef5ee9730ed"
```

Comment:
Align the arguments to `url`:
```suggestion
  url "https://github.com/open-policy-agent/gatekeeper.git",
      tag:      "v3.9.0",
      revision: "dfd2dd80734e20d30c00f7e729f03ef5ee9730ed"
```
Do we need a Git clone? Can we use a tarball?

Reply:
There is a source tarball - I ended up going this route because looking at other go based formulas, this seemed to be the pattern (kubectl, kubectx, k9s). I can certainly switch it though if that's preferred.